contrib/scripts/codepage_to_utf8.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python3
+"""
+Convert file from legacy codepage to Unicode UTF-8
+"""
+import argparse
+import os
+import sys
+
+parser = argparse.ArgumentParser(description="Convert file from legacy codepage to Unicode UTF-8",
+                                 formatter_class=argparse.RawTextHelpFormatter)
+parser.add_argument("filename", help="filename to convert")
+parser.add_argument("--cp", type=int, default=437, help="codepage to convert from (default: 437)")
+parser.add_argument("-o", "--output", type=str,
```

Comment:
Same as above, but the file type can be for writing and you can even set the encoding:

`FileType('w', encoding='UTF-8')`

Reply:
see above